File: PASCAL/Cap04/c04ex08.pas
program c04ex08;

var
  Transp: Char;

begin
  WriteLn('Escolha o tipo de transporte:');
  WriteLn;
  WriteLn('[M] - Motocicleta');
  WriteLn('[S] - Scooter');
  WriteLn;

  Write('==> ');
  ReadLn(Transp);
  WriteLn;

  if (UpCase(Transp) = 'M') or (UpCase(Transp) = 'S') then
    WriteLn('Transporte valido.')
  else
    WriteLn('Transporte invalido.');

  WriteLn;
  Write('Tecle <Enter> para encerrar... ');
  ReadLn;
end.

File: PHP/Cap04/c04ex08.php
<?php
// c04ex08.php

printf("ESCOLHA O TIPO DE TRANSPORTE:\n\n");

printf("[M] - Motocicleta\n");
printf("[S] - Scooter\n");

printf("\n==> ");
$Transp = strtoupper(trim(fgets(STDIN)));

printf("\n");

if ($Transp == 'M' || $Transp == 'S')
  printf("Transporte valido.\n");
else
  printf("Transporte invalido.\n");

printf("\n");
printf("Tecle <Enter> para encerrar... ");
trim(fgets(STDIN));

?>
